pkg/flow/flow_mem_driver.go

```diff
@@ -811,6 +811,11 @@ func (fc *FlowCollector) updateRecord(record interface{}) error {
 				if link.EndTime > 0 {
 					current.EndTime = link.EndTime
 					fc.deleteRecord(current)
+				} else {
+					if current.LinkCost == nil && link.LinkCost != nil {
+						current.LinkCost = link.LinkCost
+						fc.addRecord(current)
```

Comment:
Fairly certain this call to `fc.addRecord` is not necessary, setting LinkCost on the *LinkRecord should be sufficient.

Reply:
@c-kruse question with the skupper-network-status configmap be updated if I don't call fc.addRecord?

Reply:
@c-kruse I removed the call to fc.addRecord and I do see the configmap does appear to have the correct costs.

Reply:
@lynnemorrison I think you we're correct. It probably does happen to appear in the configmap without addRecord, but that is likely a coincidence - the collector reacting to a different event (it tends to be quite chatty.)

addRecord is a fine way to guarantee the configmap gets updated here. The way I did it for another scenario in updateRecord is by setting this updatesNetworkStatus flag: https://github.com/skupperproject/skupper/blob/92ed34c6087e1c8cef33f97807c0c1f8abee592c/pkg/flow/flow_mem_driver.go#L1228-L1230